### PR TITLE
Bump otel versions to 1.3.0 (and updage changelog)

### PR DESCRIPTION
**Pull request by mateuszrzeszutek**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,10 +10,21 @@ and this repository adheres to [Semantic Versioning](https://semver.org/spec/v2.
 
 ### General
 
-- First stable release of the Splunk Distribution of OpenTelemetry Java.
+- OpenTelemetry Java SDK and OpenTelemetry Instrumentation for Java dependencies were updated to version 1.3.0.
 
 ## v1.0.0 - 2021-06-02
```
Comment (iNikem) [inline CHANGELOG.md:15]: This should be a new section for 1.1.0 release, right?

File: gradle/dependencies.gradle [hunk 2]
```diff
@@ -4,8 +4,8 @@ ext {
   versions = [
       opentelemetry              : snapshot ? "1.4.0-SNAPSHOT" : "1.3.0",
       opentelemetryAlpha         : snapshot ? "1.4.0-alpha-SNAPSHOT" : "1.3.0-alpha",
-      opentelemetryJavaagent     : snapshot ? "1.3.0-SNAPSHOT" : "1.3.0-SNAPSHOT",
-      opentelemetryJavaagentAlpha: snapshot ? "1.3.0-alpha-SNAPSHOT" : "1.3.0-alpha-SNAPSHOT",
+      opentelemetryJavaagent     : snapshot ? "1.3.0-SNAPSHOT" : "1.3.0",
+      opentelemetryJavaagentAlpha: snapshot ? "1.3.0-alpha-SNAPSHOT" : "1.3.0-alpha",
```
Comment (iNikem) [inline gradle/dependencies.gradle:8]:
```suggestion
      opentelemetryJavaagent     : snapshot ? "1.4.0-SNAPSHOT" : "1.3.0",
      opentelemetryJavaagentAlpha: snapshot ? "1.4.0-alpha-SNAPSHOT" : "1.3.0-alpha",
```

**Review by iNikem — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (mateuszrzeszutek) [reply CHANGELOG.md:15]: There will be, the `./pre-release.sh` adds it automatically - I'll run it and post a separate PR

File: gradle/dependencies.gradle (hunk 2 above)
Comment (mateuszrzeszutek) [reply gradle/dependencies.gradle:8]: ouch

**Review by iNikem — APPROVED** [review]

**Pull request merged**